x/protorev/keeper/developer_fees_test.go

```diff
@@ -0,0 +1,168 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v13/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v13/x/protorev/types"
+)
+
+// TestSendDeveloperFeesToDeveloperAccoutn tests the SendDeveloperFeesToDeveloperAccount function
+func (suite *KeeperTestSuite) TestSendDeveloperFeesToDeveloperAccount() {
+	cases := []struct {
+		description   string
+		malleate      func()
```

Comment:
What does malleate mean?

Reply:
malleate i.e. applying some changes to state using a wrapper function.

Reply:
might be more straightforward to just use alterState or something along those lines. will update that.

Reply:
updated to use `alterState`